services/brig/src/Brig/Data/Client.hs

```diff
@@ -247,6 +266,31 @@ claimPrekey u c =
       ind <- liftIO $ randomRIO (0, length pks' - 1)
       return $ atMay pks' ind
 
+lookupMLSPublicKey :: UserId -> ClientId -> SignatureSchemeTag -> AppIO r (Maybe LByteString)
```

Comment:
Where possible, I'd prefer these data functions to run in a `MonadClient m` rather than `AppIO r` (with a `MonadReader` constraint where needed).